podpac/core/data/rasterio_source.py

```diff
@@ -140,10 +155,65 @@ def get_data(self, coordinates, coordinates_index):
         data.data.ravel()[:] = raster_data.ravel()
         return data
 
+    def get_data_overviews(self, coordinates, coordinates_index):
+        # Figure out how much coarser the request is than the actual data
+        reduction_factor = np.inf
+        for c in ["lat", "lon"]:
```

Comment:
I'm assuming uniform lat/lon resolution

Reply:
What do you mean here? You are assuming that the requested coordinates are uniform lat/lon, or that the overview coordinates are? Can we just check that and raise a RuntimeError if they aren't uniform?